zib/StructureDefinitions/zib-AnatomicalLocation.xml

```diff
@@ -0,0 +1,92 @@
+  <title value="zib AnatomicalLocation" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="An anatomical location specifies the location (e.g. foot) and laterality (e.g. left) of a body part." />
+  <purpose value="This CodeableConcept resource represents Location concept of the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) AnatomicalLocation v1.0.3 (2024)](https://zibs.nl/wiki/AnatomicalLocation-v1.0.3(2024EN)). The zib AnatomicalLocation is mapped to both a profile on BodyStructure (&lt;http://nictiz.nl/fhir/StructureDefinition/zib-AnatomicalLocation-BodyStructure&gt;) and this CodeableConcept profile. The CodeableConcept handles the representation of the location, while the BodyStructure profile covers both location and laterality.&#xD;&#xA;&#xD;&#xA;Profilers can use this profile in places where zib AnatomicalLocation is instantiated." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-anatomicallocation-v1.0.3-2024EN" />
+    <uri value="https://zibs.nl/wiki/AnatomicalLocation-v1.0.3(2024EN)" />
+    <name value="zib AnatomicalLocation-v1.0.3(2024EN)" />
+  </mapping>
+  <kind value="complex-type" />
+  <abstract value="true" />
+  <type value="CodeableConcept" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/CodeableConcept" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="CodeableConcept">
+      <path value="CodeableConcept" />
+      <short value="Location" />
+      <definition value="Localisation on/in the body." />
+      <comment value="In cases where only the location is required, or where laterality is embedded within the location code, this CodeableConcept can be used within the resource. However, when laterality needs to be explicitly recorded, the BodyStructure resource should be used, referenced through the BodyStructure Reference within this CodeableConcept. If the BodyStructure reference is used, this CodeableConcept should either remain empty or align with the `BodyStructure.location` element.&#xD;&#xA;&#xD;&#xA;**Note:** Receiving systems must support both this CodeableConcept and the BodyStructure resource, as either mechanism may be employed. The rationale for this is that, in many cases, only the location will be required, simplifying the FHIR recording process for sending systems." />
```

Comment:
Please keep in mind dat this is the comment that ends up the profile when using this data type profile. It should be targeted at users of the profile.

Reply:
This is text is aimed towards implementers of this profile. Not sure what of it is not. Could you highlight what you think is wrong?

Reply:
maybe the 'required' part can be adjusted

Reply:
Slightly adjusted.

Reply:
So, I'm trying to imagine reading this text on e.g. `Observation.bodySite`.

> A deviation from our general profiling has been implemented in this profile.

From an implementer perspective, it might seem strange to see the words "our" and "profile" here. I think it would be more logical to have something like:

> In this case, slicing is not according to the Nictiz profiling guidelines.

Some other suggestions:

> Slicing has been applied to `CodeableConcept.coding` to enable the use of the `BodyStructure` reference extension without requiring a populated coded value. This approach facilitates scenarios where only the Laterality element needs to be conveyed.

Could be adjusted to:

> Slicing has been applied to `CodeableConcept.coding` instead of `CodeableConcept` directly to enable the use of the `BodyStructure` reference extension without requiring a coded value. This approach facilitates scenarios where only the zib Laterality concept needs to be conveyed.

And I think the following is not true:

> The decision was made due to the Location concept’s required binding strength. If the required binding were applied on the CodeableConcept, it would mandate a coded value. This slicing approach ensures flexibility while maintaining compliance with the required binding constraints for Location.

This way of slicing doesn't comply with the required binding constraint, as it makes the use codes from this value set completely optional.

Reply:
Thanks those are indeed good textual improvements.

Regarding "This slicing approach ensures flexibility while maintaining compliance with the required binding constraints for Location." I was thinking of the level of where the mapping is placed, namely a .coding element with 0..1 with a required binding indicating that a code needs to be chosen from those valuesets. Together with the invariant it gets very close to a similar effect as a required binding strenght.  

But reading it again, I also agree that the text needs updating.

Reply:
Outdated and removed with latest commit.